src/components/Avatar/AvatarIcon.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+
+const propTypes = {
+    tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+    className: PropTypes.string,
+    link: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+}
+
+const defaultProps = {
+    tag: 'div',
+    anchor: 'a',
```

Comment:
If the criteria to change from `div` to `a` is the `href` prop, then we can simply use that to change, without an additional `anchor` prop.